README.md

```diff
@@ -19,19 +25,19 @@ npm run certificate
 npm run start
 ```
 
-Now you will be able to visit the web-based calendar UI under https://localhost:8080/
-
-The API documentation is available at https://localhost:8080/docs.
+Now you will be able to visit the web-based calendar interface at https://localhost:8080/
 
```

Comment:
I think you should mention the API here, since it is a beautiful page ;)

Reply:
Steht ganz unten. Ist ja für den Endnutzer nicht so relevant.